README.md

```diff
@@ -512,10 +533,46 @@ function prefetch () {
 And use it when you need to preload the **resources** (for example when [hovering](https://github.com/GoogleChromeLabs/quicklink) [a](https://github.com/guess-js/guess) [link](https://instant.page)).  
 Together with techniques like [page prefetching](https://nextjs.org/docs#prefetching-pages) in Next.js, you will be able to load both next page and data instantly.
 
+### Request Deduplication
+
+SWR provides request deduplication by default. So, if you call the hook with same key multiple times, it will return the cached value ie. requests will not be duplicated.
+Only one request will be made and once the promise is fulfilled, it will update both components.
+Here, 'api/user' is the duplicate key
+
+```js
+import useSWR from 'swr'
+
+function UserProfileName() {
+  const { data, error } = useSWR('/api/user', fetcher)
+
+  if (error) return <div>failed to load</div>
+  if (!data) return <div>loading...</div>
+  return <p>Name: {data.name}!</p>
+}
+
+function UserProfileAvatar() {
+  const { data, error } = useSWR('/api/user', fetcher)
+
+  if (error) return <div>failed to load</div>
+  if (!data) return <div>loading...</div>
+  return <img src={data.avatarUrl} alt="Profile image" />
+}
+
+export default function App() {
+  return (
+    <div>
+      <UserAvatar />
+      <UserProfile />
+    </div>
+  )
+}
+```
```

Comment:
At this point I think we should explain the `dedupingInterval` option, so you can customize how the deduping works.

Reply:
Thanks for the suggestion. Let me add that

Reply:
Done. @pacocoursey  Please check